pkg/deploy/generator/scripts/util-services.sh

```diff
@@ -292,17 +301,218 @@ ExecStart=/usr/bin/podman run \
+    # shellcheck disable=SC2034
+    local -r aro_mise_service_file='[Unit]
+After=network-online.target
+Wants=network-online.target
+StartLimitIntervalSec=0
+[Service]
+RestartSec=1s
+EnvironmentFile=/etc/sysconfig/aro-mise
+ExecStartPre=-/usr/bin/docker rm -f %N
+ExecStart=/usr/bin/docker run \
+  -p 5000:5000 \
+  -v /app/mise/appsettings.json:/app/appsettings.json:z \
+  --hostname %H \
+  --name %N \
+  --network=${PODMAN_NETWORK} \
+  --rm \
+  $MISEIMAGE
+ExecStop=/usr/bin/docker stop %N
+Restart=always
+RestartSec=3
+StartLimitInterval=0
+[Install]
+WantedBy=multi-user.target'
+
+    write_file aro_mise_service_filename aro_mise_service_file true
+}
+# configure_service_aro_otel_collector
+# args:
+# 1) image - nameref, string; OTEL container image
+# 2) network - nameref, string; podman network name to be attached
+configure_service_aro_otel_collector() {
+    local -n image="$1"
+    local -n network="$2"
+    log "starting"
+    log "Configuring aro-otel-collector service"
+
+    # shellcheck disable=SC2034
+    local -r aro_otel_collector_service_conf_filename='/etc/sysconfig/aro-otel-collector'
+    # shellcheck disable=SC2034
+    local -r aro_otel_collector_service_conf_file="GOMEMLIMIT=1000MiB
+OTELIMAGE='$image'
+PODMAN_NETWORK='$network'"
+
+    write_file aro_otel_collector_service_conf_filename aro_otel_collector_service_conf_file true
+
+    mkdir -p /app/otel
+    # shellcheck disable=SC2034
+    local -r aro_otel_collector_appconfig_filename='/app/otel/config.yaml'
+    # shellcheck disable=SC2034
+    local -r aro_otel_collector_appconfig_file='receivers:
+  httpcheck:
+    targets:
+    # MISE Endpoints
+      - endpoint: http://aro-mise:5000/healthz
+        method: GET
+      - endpoint: http://aro-mise:5000/readyz
+        method: GET
+    # OTELs own Endpoints
+      - endpoint: http://aro-otel-collector:13133/healthz
+        method: GET
+      - endpoint: http://aro-otel-collector:13133/readyz
+        method: GET
+    collection_interval: 20s
+processors:
+  batch:
+extensions:
+  health_check:
+    endpoint: "aro-otel-collector:13133"
+exporters:
+  otlp:
+    endpoint: mdm:4317
+    tls:
+      insecure: true
+service:
+  extensions: [health_check]
+  pipelines:
+    metrics:
+      receivers: [httpcheck]
+      processors: [batch]
+      exporters: [otlp]'
+
+    write_file aro_otel_collector_appconfig_filename aro_otel_collector_service_conf_file true
+
+    # shellcheck disable=SC2034
+    local -r aro_otel_collector_service_filename='/etc/systemd/system/aro-otel-collector.service'
+    # shellcheck disable=SC2034
+    local -r aro_otel_collector_service_file='[Unit]
+After=network-online.target
+Wants=network-online.target
+StartLimitIntervalSec=0
+[Service]
+RestartSec=1s
+EnvironmentFile=/etc/sysconfig/aro-otel-collector
+ExecStartPre=-/usr/bin/docker rm -f %N
+ExecStart=/usr/bin/docker run \
+  --hostname %H \
+  --name %N \
+  --rm \
+  --network=${PODMAN_NETWORK} \
```

Comment:
Can you add a commit to make this change for the other service files? That way it's all consistent.
I'm fine with putting the network in the systemd env file, but it should be consistent with the rest of the services.